src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -295,3 +295,204 @@ and if they have a value depending on the event origin.
 | Y
 | Y
 |=====
+
+[[microsoft_teams_configuration]]
+==== Microsoft Teams notification method
+
+ProActive provides a ready to use Third-Party script that allows users to send notifications to multiple Teams channels.
```

Comment:
maybe add something like:  This ensures that critical alerts and updates reach users directly in their channels, streamlining communication.

Reply:
Sure I can add something like that but I'll do it in the User doc then, here the focus is on configuration.